servers/rendering/renderer_rd/renderer_storage_rd.h

```diff
@@ -39,9 +41,9 @@
 #include "servers/rendering/renderer_rd/shaders/giprobe_sdf.glsl.gen.h"
 #include "servers/rendering/renderer_rd/shaders/particles.glsl.gen.h"
 #include "servers/rendering/renderer_rd/shaders/particles_copy.glsl.gen.h"
+#include "servers/rendering/renderer_rd/shaders/skeleton.glsl.gen.h"
 #include "servers/rendering/renderer_scene_render.h"
 #include "servers/rendering/rendering_device.h"
-
```

Comment:
Why? That's ugly :)

Reply:
I don't really see this when I write the code to be honest, I though these things were automatically fixed

Reply:
I don't think there's any option to force adding a space between includes and class definition in `clang-format` sadly.